.lintrunner.toml

```diff
@@ -254,17 +254,20 @@ command = [
 [[linter]]
 code = 'CLANGTIDY'
 include_patterns = [
+    'aten/src/ATen/core/*.cpp',
     'c10/**/*.cpp',
     'torch/csrc/**/*.cpp',
 ]
 exclude_patterns = [
     # The negative filters below are to exclude files that include onnx_pb.h or
     # caffe2_pb.h, otherwise we'd have to build protos as part of this CI job.
     # CUDA files are also excluded.
+    '**/build/**',
     '**/fb/**',
     '**/*pb.h',
     '**/*CUDA*',
     '**/cuda/*pp',
+    'aten/src/ATen/core/*test.cpp',
```

Comment:
Is this glob a little overbroad? Do all the test files use protobuff? Or include a header file that does?

Reply:
I want to exclude 2 tests, the one using protobuf and another with std::move